server/events/apply_command_runner.go

```diff
@@ -80,7 +80,7 @@ func (a *ApplyCommandRunner) Run(ctx *command.Context, cmd *command.Comment) {
 		return
 	}
 
-	statusId, err := a.commitStatusUpdater.UpdateCombined(context.TODO(), baseRepo, pull, models.PendingCommitStatus, cmd.CommandName(), "")
+	statusId, err := a.commitStatusUpdater.UpdateCombined(context.TODO(), baseRepo, pull, models.PendingCommitStatus, cmd.CommandName(), "", "")
```

Comment:
as a reader it's confusing to see `"", ""` and this function has 7 parameters.  Feels like either we've gotten to a point where we need to combine args, or assess whether this abstraction is even useful anymore.

 I'm fine with this for now since this code will go away, but I'm calling it out as a learning thing.

Reply:
Yup, I also noticed this wasn't ideal but I chose to go this simpler route becuase we will eventually remove this. Thanks for the callout though :)